### PR TITLE
Quote the username sent to idm as a query when calling discharge_token.

### PR DESCRIPTION
**Pull request by frankban**
Also bump the version up.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/169/
Test FAILed.

**Comment by mhilton** [comment]
:+1: when the tests pass

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 setup(
     name='theblues',
-    version='0.3.7',
+    version='0.3.8',
```
Comment (bac) [inline setup.py:21]: Hmm, we never added 0.3.7 to blues-browser.

**Review by bac — APPROVED** [review]

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/170/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/171/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/172/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/173/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/174/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/175/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/176/
Test PASSed.

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/theblues-lib-merge

**Pull request merged**